OMCompiler/Compiler/SimCode/SimCodeUtil.mo

```diff
@@ -8228,55 +8228,32 @@ end derVarFromStateVar;
 public function simVarString
```

Comment:
We should have a short doc string that explains the purpose of the function.

Reply:
Absolutely!